lib/api/completeMultipartUpload.js

```diff
@@ -142,21 +142,13 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 key: item.key,
                 ETag: `"${item.value.ETag}"`,
                 size: item.value.Size,
-                // If the stored part has a partLocations.key property,
-                // use that for the actual key.  If not, it means we
-                // have an object using an md-model-version prior to 2 so
-                // item.vaule.partLocations will just contain the key
-                location: item.value.partLocations.key ?
-                    item.value.partLocations.key : item.value.partLocations,
-                // If the stored part has a partLocations.dataStoreName
-                // property use that for the actual dataStoreName.
-                // If not, default to sproxyd
-                dataStoreName: item.value.partLocations.dataStoreName ?
-                    item.value.partLocations.dataStoreName : 'sproxyd',
-                sseAlgorithm: item.value.partLocations.sseAlgorithm,
-                sseMasterKeyId: item.value.partLocations.sseMasterKeyId,
-                sseCryptoScheme: item.value.partLocations.sseCryptoScheme,
-                sseCipheredDataKey: item.value.partLocations.sseCipheredDataKey,
+
+                // BREAKING CHANGE -- NO LONGER SUPPORT PART DATA STRUCTURE
```

Comment:
Since you assessed that no-one has that issue, can we get rid of the comment here? The commit message can give enough context by itself.